### PR TITLE
Feature/plug in create account

### PR DESCRIPTION
**Pull request by cammellos**

### CODE REVIEW

File: modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java [hunk 1]
```diff
@@ -323,6 +323,20 @@ public void prepareDirAndUpdateConfig(final String keyUID, final String config,
         callback.invoke(finalConfig);
     }
 
+    @ReactMethod
+    public void createAccountAndLogin(final String createAccountRequest) {
```
Comment (flexsurfer) [inline modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java:327]: just wondering why we need a new endpoint?

Comment (cammellos) [reply modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java:327]: Simplifying,
basically now we need to call to create an account:
- OpenAccounts
- InitKeystore
- MultiaccountGenerateAndDeriveAddress
- MultiaccountStoreAccount
- SaveAccountAndLogin

Generate and derive addresses takes all the `paths` for wallets etc,
And save account and login takes the whole NodeConfig & Settings.

A lot of this I think it should be in status-go, so I am providing a convenience endpoint to create a multiaccount that should get rid of having to make all those calls above and just make a simple one, ideally specifying the least amount of arguments that we need.

What do you think?

Comment (flexsurfer) [reply modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java:327]: i think thats great :)

**Comment by cammellos** [comment]
closing as already merged

**Pull request closed without merging**